all/Dockerfile

```diff
@@ -164,10 +220,6 @@ RUN wget https://dl.bintray.com/sbt/native-packages/sbt/0.13.9/sbt-0.13.9.tgz \
   && rm sbt-0.13.9.tgz
 RUN ln -s /usr/local/share/sbt/0.13.9/sbt/bin/sbt /usr/local/bin/sbt
 WORKDIR /root/tmp
-RUN sbt -sbt-version 0.13.5
-RUN sbt -sbt-version 0.13.6
-RUN sbt -sbt-version 0.13.7
-RUN sbt -sbt-version 0.13.8
 RUN sbt -sbt-version 0.13.9
```

Comment:
There are no challenges which test is written by Scala. Right?
If so, this change is **almost** no problem. (
The reason why I said `almost` is because there might be some ChellengeResult which written is Scala and old sbt.
If so, running that code on whale(or orca) will take long long time to download sbt.

Also, I found that cli-template for scala doesn't specify sbt.version.(build.properties file is absent.)

https://github.com/code-check/cli-template-scala

It might be better to specify same version which installed on env.
(However specify the version 0.13.x is too lagacy... Hopefully it is better to use sbt 1.x)

Reply:
As I discussed in offline, I'll keep only one version with 0.13.x